main.py

```diff
@@ -1,27 +1,59 @@
-"""import all models""" # pylint:
+"""import all models"""  # pylint:
```

Comment:
I guess module docstring is something that describes what module is doing, not several lines of code below